applications/FluidDynamicsApplication/custom_elements/qsvms_dem_coupled.h

```diff
@@ -0,0 +1,379 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Joaquin Gonzalez-Usua
+//
+
+#ifndef KRATOS_QSVMS_DEM_COUPLED_H
+#define KRATOS_QSVMS_DEM_COUPLED_H
+
+// System includes
+#include <string>
+#include <iostream>
+
+#include "includes/checks.h"
+#include "includes/element.h"
+#include "includes/serializer.h"
+#include "geometries/geometry.h"
+
+#include "includes/cfd_variables.h"
+#include "custom_elements/fluid_element.h"
+#include "custom_elements/qs_vms.h"
+#include "fluid_dynamics_application_variables.h"
```

Comment:
```suggestion
// System includes
#include <string>
#include <iostream>

// External includes

// Project includes
#include "includes/checks.h"
#include "includes/element.h"
#include "includes/serializer.h"
#include "geometries/geometry.h"
#include "includes/cfd_variables.h"

// Application includes
#include "qs_vms.h"
```
I think that it should work in this way.